fire-hose-management-frontend/src/components/Departments/EditDepartmentOverlay.jsx

```diff
@@ -0,0 +1,179 @@
+import {makeStyles} from "@material-ui/core/styles";
+import React, {useState} from "react";
```

Comment:
Codacy has a fix for the issue: [Replace `useState` with `·useState·`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
import React, { useState } from "react";
```